src/plugins/workspace/public/components/workspace_creator/workspace_creator.test.tsx

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import { WorkspaceCreator } from './workspace_creator';
+import { PublicAppInfo } from 'opensearch-dashboards/public';
+import { fireEvent, render, waitFor } from '@testing-library/react';
+import * as opensearchReactExports from '../../../../opensearch_dashboards_react/public';
+import { BehaviorSubject } from 'rxjs';
+import { WorkspaceFormSubmitData } from './workspace_form';
+
+jest.mock('../../../../opensearch_dashboards_react/public', () => ({
+  ...jest.requireActual('../../../../opensearch_dashboards_react/public'),
+  __esModule: true,
+}));
```

Comment:
If we return the actual implementation of the target module, why do we doMock on the path?